sympy/functions/special/error_functions.py

```diff
@@ -128,6 +128,8 @@ def eval(cls, arg):
         if isinstance(arg, erfcinv):
             return S.One - arg.args[0]
 
+        # Unlikely to happend since erf2inv(0, x) is evaluated to
+        # erfinv(x)
         if isinstance(arg, erf2inv) and arg.args[0] is S.Zero:
             return arg.args[1]
```

Comment:
If this isn't covered you could test it with erf2inv(0, x, evaluate=False).

Reply:
The problem is that when I then do `erf(erf2inv(0, x, evaluate=False))` it keeps it on that form. And if I evaluate it (which I do not for sure known how to...), I am still not sure in which order it is evaluated...

Reply:
I'm not sure what you mean. It seems to evaluate automatically here:
```julia
In [2]: erf2inv(0, x, evaluate=False)                                                                                             
Out[2]: erf2inv(0, x)

In [3]: erf(_)                                                                                                                    
Out[3]: x
```

Reply:
OK! I strongly recall trying it before your suggestion, but I may have done something different... I'll give it a new go.

Reply:
`erf(erf2inv(0, x, evaluate=False))` does not evaluate, despite the `evaluate=False` clause being on the innermost expression. But maybe that is how it is supposed to work.

Reply:
Gah! Forgot to add the test... Will do it after I see how the coverage turns out...

Reply:
It does evaluate for me on master:
```julia
In [2]: x = Symbol('x')                                                                                                           

In [3]: erf(erf2inv(0, x, evaluate=False))                                                                                        
Out[3]: x
```

Reply:
Weird! I also run master...

Reply:
I just tried this on SymPy Live and I get the same:
https://live.sympy.org/

I also tried with both Python 2.7 and 3.7 and SymPy master locally.

Are you sure you tested with master? It should evaluate because of the code we are commenting on...

Reply:
I have a Python session where it doesn't work (and I think that is different from the one where it earlier didn't work, I tend to restart them quite often). Not sure why and how though... Seems to work after starting a new one though and on my Linux machine. Yes, was running master (or rather this branch, but that is basically master).